src/WCSimPrimaryGeneratorMessenger.cc

```diff
@@ -55,12 +55,12 @@ WCSimPrimaryGeneratorMessenger::WCSimPrimaryGeneratorMessenger(WCSimPrimaryGener
   isotopeCmd->SetParameter(param);
   
   radonScalingCmd = new G4UIcmdWithAString("/mygen/radon_scaling",this);
-  radonScalingCmd->SetGuidance("Select scalling scenario");
+  radonScalingCmd->SetGuidance("Select scalling scenario, if scenario 0 is selected, Bi214 are generated uniformly");
   radonScalingCmd->SetGuidance("[usage] /mygen/radon SCENARIO ");
-  radonScalingCmd->SetGuidance("     SCENARIO : A, B, C");
-  radonScalingCmd->SetCandidates("A B C");
+  radonScalingCmd->SetGuidance("     SCENARIO : 0, A, B");
+  radonScalingCmd->SetCandidates("0 A B");
   param = new G4UIparameter("SCENARIO",'s',true);
-  param->SetDefaultValue("C");
+  param->SetDefaultValue("B");
```

Comment:
```suggestion
  param->SetDefaultValue("A");
```
According to the mac file, A should be default for now